app/controllers/llm_api_keys_controller.rb

```diff
@@ -5,4 +5,21 @@ class LlmApiKeysController < ApplicationController
   def index
     @llm_api_keys = current_user.llm_api_keys
   end
+
+  # POST /user/:user_id/llm_api_keys
+  def create
+    llm_api_key = current_user.llm_api_keys.build(llm_api_key_params)
+    llm_api_key.save!
```

Comment:
```suggestion
    llm_api_key = current_user.llm_api_keys.create!(llm_api_key_params)
```

build と save! を組み合わせなくても、create! で一度に書けるのでシンプルになると思います。

もしllm_api_key が不要であれば消しておいてください。